css/interface.css

```diff
@@ -107,6 +107,19 @@
   margin: 0;
 }
 
+.bundleId-ast-text,
```

Comment:
`ast`?

- appStore: `appStore`
- ent: `enterprise`
- uns: `unsigned`

Reply:
@squallstar It was called like this before. Should I change the name?

Reply:
yes please, not sure what `ast` meant but if it's related to Google Play / App Store then we should use `store` for consistency.